src/app/personside/infotabs/saksoversikt/saksdokumenter/DokumentListeElement.tsx

```diff
@@ -214,22 +218,24 @@ class DokumentListeElement extends React.Component<Props> {
                 }
                 ref={this.dokumentRef}
                 valgt={this.props.dokument === this.props.valgtDokument && this.props.visDokument}
-                klikkbar={kanVises}
+                klikkbar={tilgangTilHoveddokument}
             >
-                <IkonWrapper>{dokumentIkon(kanVises)}</IkonWrapper>
+                <IkonWrapper>{dokumentIkon(this.harTilgangTilJournalpost(dokumentMetadata))}</IkonWrapper>
                 <InnholdWrapper>
                     <UUcustomOrder>
                         <div ref={this.hoveddokumentLinkRef} className="order-second">
                             <LenkeKnapp
-                                aria-disabled={!kanVises}
-                                onClick={() => this.visDokumentHvisTilgang(dokument.hoveddokument)}
+                                aria-disabled={!tilgangTilHoveddokument}
```

Comment:
dette blir kanskje feil, for kan vel åpnes nå selv om man ikke har tilgang til hoveddokument..?

Reply:
Njei, dette er vel den lenken som går til hoveddokument? Skal man kunne åpne den uten tilgang til hoveddokument?

Reply:
aha, my bad